secant/Features/ImportWallet/ImportWalletStore.swift

```diff
@@ -79,7 +79,7 @@ struct ImportWalletReducer: ReducerProtocol {
                 
             case .birthdayInputChanged(let redactedBirthday):
                 state.birthdayHeight = redactedBirthday
-                if let birthdayHeight = BlockHeight(state.birthdayHeight.data), birthdayHeight >= zcashSDKEnvironment.defaultBirthday {
```

Comment:
@LukasKorba one of the tests fail because it expects this to have a value and not be `nil`. I don't understand the role of `defaultBirthday` and that this If statement is meant to check

Reply:
I assume you mean that the default block height is the minimum blockheight that would make a birthday valid in terms of the capabilities of this wallet application. If so, I changed this default block to sapling activation for the corresponding network

Reply:
> I assume you mean that the default block height is the minimum blockheight that would make a birthday valid in terms of the capabilities of this wallet application. If so, I changed this default block to sapling activation for the corresponding network

exactly